src/redux/protocolContracts.ts

```diff
@@ -0,0 +1,119 @@
+    superfluidLoaderv1: "0x74d860243Ff08A243d5485899f343117EbDa6eA8",
+    TOGA: "0xa54FC15FC75693447d70a57262F37a70B614721b",
+  },
+  kovan: {
+    resolver: "0x851d3dd9dc97c1df1DA73467449B3893fc76D85B",
+    host: "0xF0d7d1D47109bA426B9D8A3Cde1941327af1eea3",
+    CFAv1: "0xECa8056809e7e8db04A8fF6e4E82cD889a46FE2F",
+    IDAv1: "0x556ba0b3296027Dd7BCEb603aE53dEc3Ac283d2b",
+    superTokenFactory: "0xF5F666AC8F581bAef8dC36C7C8828303Bd4F8561",
+    superfluidLoaderv1: "0x9F73208FAFf97465CEC27fd93F10EB3D78289339",
+  },
+  rinkeby: {
+    resolver: "0x659635Fab0A0cef1293f7eb3c7934542B6A6B31A",
+    host: "0xeD5B5b32110c3Ded02a07c8b8e97513FAfb883B6",
+    CFAv1: "0xF4C5310E51F6079F601a5fb7120bC72a70b96e2A",
+    IDAv1: "0x32E0ecb72C1dDD92B007405F8102c1556624264D",
+    superTokenFactory: "0xd465e36e607d493cd4CC1e83bea275712BECd5E0",
+    superfluidLoaderv1: "0x70A755e96d321dccd26111233a297095910784C7",
+  },
+  ropsten: {
+    resolver: "0x3b44e06D96BcA9412CBc23F80F41B9e30933571a",
+    host: "0xF2B4E81ba39F5215Db2e05B2F66f482BB8e87FD2",
+    CFAv1: "0xaD2F1f7cd663f6a15742675f975CcBD42bb23a88",
+    IDAv1: "0xAD1e87F0C74341ecAFc1d27349dD6e650f5bAdD7",
+    superTokenFactory: "0x6FA165d10b907592779301C23C8Ac9d1F79ca930",
+    superfluidLoaderv1: "0x23e9A1D629AC39e60c60C7592EB12fB8247Bd52A",
+  },
+  mumbai: {
+    resolver: "0x8C54C83FbDe3C59e59dd6E324531FB93d4F504d3",
+    host: "0xEB796bdb90fFA0f28255275e16936D25d3418603",
+    CFAv1: "0x49e565Ed1bdc17F3d220f72DF0857C26FA83F873",
+    IDAv1: "0x804348D4960a61f2d5F9ce9103027A3E849E09b8",
+    superTokenFactory: "0x200657E2f123761662567A1744f9ACAe50dF47E6",
+    superfluidLoaderv1: "0x0d56ED56b63382B0FC964490feB9AE438B6B4b79",
+  },
+  "optimism-kovan": {
+    resolver: "0x218B65780615Ff134f9Ad810CB98839534D3C0D6",
+    host: "0x74b57883f8ce9F2BD330286E884CfD8BB24AC4ED",
+    CFAv1: "0x523481a8a5455efB1F301C6e00B5525Eb12a51da",
+    IDAv1: "0x98E5E5d915Bf79ceeF02c72D1bf8f5b26f0bcBaA",
+    superTokenFactory: "0xB5f0501908ca8A99fD31bEFCAc1cA458F3588671",
+    superfluidLoaderv1: "0x777Be25F9fdcA87e8a0E06Ad4be93d65429FCb9f",
+  },
+  "arbitrum-rinkeby": {
+    resolver: "0xa2C0C70A1E922f5f060ec20EE3aF002C163b4567",
+    host: "0xE01F8743677Da897F4e7De9073b57Bf034FC2433",
+    CFAv1: "0xcA0569223e56B479FaC6d554FA0D3867A4384657",
+    IDAv1: "0x038405394F2B791F71c2C68d36dc91C568DfAB05",
+    superTokenFactory: "0xc11eC618c1d559705E853741e366663Fe9302362",
+    superfluidLoaderv1: "0x777Be25F9fdcA87e8a0E06Ad4be93d65429FCb9f",
+  },
+  "avalanche-fuji": {
+    resolver: "0x141920741bC45b962B59c833cd849bA617F7ef38",
+    host: "0xf04F2C525819691ed9ABD3D2B7109E1633795e68",
+    CFAv1: "0xED74d30B8034152b0638CB03cc5c3c906dd1c482",
+    IDAv1: "0x997d745884F54a93E6662b055c5e6c09F688718b",
+    superTokenFactory: "0xA25dbEa94C5824892006b30a629213E7Bf238624",
+    superfluidLoaderv1: "0x777Be25F9fdcA87e8a0E06Ad4be93d65429FCb9f",
+  },
+} as NetworkContracts;

```

Comment:
In places like this, I'd rather do: `const protocolContracts: NetworkContracts = { ...`